README.md

```diff
@@ -134,17 +134,25 @@ Furthermore, there will be a description on what CRD they monitor and a general
 
 ### Webhook
 
-The mutating webhook works only on resources that have Keptn annotations.
-The mutation consists in changing the scheduler used for the deployment with the Keptn Scheduler.
-The webhook should be as fast as possible and should not create/change any resource.
+Annotating a namespace subjects it to the effects of the mutating webhook:
 
+```
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: podtato-kubectl
+  annotations:
+    keptn.sh/lifecycle-controller: "enabled"  # this lines tells the webhook to handle the namespace
+```
+However, the mutating webhook will modify only resources in the annotated namespace that have Keptn annotations.
 When the webhook receives a request for a new pod, it will look for the following annotations:
```

Comment:
it looks only for workload annotations, if version or app annotations do not exist, it will add them